include/operations/blas_operators.hpp

```diff
@@ -26,11 +26,12 @@
 #ifndef BLAS_OPERATORS_HPP
 #define BLAS_OPERATORS_HPP
 
+#include <CL/sycl.hpp>
+#include <climits>
 #include <iostream>
+#include <operations/blas_constants.hpp>
 #include <stdexcept>
 #include <vector>
-#include <operations/blas_constants.hpp>
-#include <CL/sycl.hpp>
```

Comment:
In your clang formatter enable --Style==Google in order to prevent the reordering of the include files